core/templates/pages/landing-pages/topic-landing-page/topic-landing-page.controller.ts

```diff
@@ -48,6 +48,9 @@ angular.module('oppia').directive('topicLandingPage', [
             TOPIC_LANDING_PAGE_DATA) {
           var ctrl = this;
           var pathArray, topic, topicData, landingPageData, assetsPathFormat;
+          const tagLines = {
+            Fractions: 'Add, Subtract, Multiply and Divide'
```

Comment:
Can we move this to `TOPIC_LANDING_PAGE_DATA`?

Reply:
Do we need these tagLines for Ratios pages as well?

Reply:
> Can we move this to `TOPIC_LANDING_PAGE_DATA`?

Done.

> Do we need these tagLines for Ratios pages as well?

Yes, we do. I think @Danialp87 will provide that and we can then add them for the ratio page and the negative numbers page.